scripts/zkai_claim_audit_comparison_artifacts_gate.py

```
@@ -0,0 +1,946 @@
+
+from __future__ import annotations
+
+import argparse
+import copy
+import csv
+import hashlib
+import io
+import json
+import pathlib
+import sys
+from typing import Any, Callable
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)